### DISCUSSION TITLE
Attributes on many-to-many relations

### DISCUSSION THREAD

**Discussion by muety** [pocketbase/pocketbase#4037] [Q&A] [answered]
Is there a way (preferable even using the admin UI) to have attributes defined on a many-to-many relation? For example, consider an `Order` model and a `Product` model. An order can consist of many products, a product can be part of many orders. In addition, I might - for example - want to store the number of instances of the same product in that order, that is, a simple integer attached to the `Order <-> Product` relation.

**Comment by ganigeorgiev** [comment, marked as answer]
Not directly. For such cases you can use a more traditional SQL approach and create a associative/junction collection, aka. a helper collection (eg. `orderProduct`) that contains 2 single `relation` fields referencing your order and product + any additional meta fields you many need.
